### PR TITLE
feat: using rust's built-in alignment to place spaces we want

### PR DESCRIPTION
**Pull request by levisyin**
close [#182](https://github.com/FuelLabs/fuelup/issues/182)

This PR added a function `pad_spaces(num: usize) -> String` to conveniently input the number of spaces we want.

### CODE REVIEW

File: src/ops/fuelup_check.rs [hunk 1]
```diff
@@ -60,18 +60,18 @@ fn check_plugin(plugin_executable: &Path, plugin: &str, latest_version: &Version
             match output.split_whitespace().last() {
                 Some(v) => {
                     let version = Version::parse(v)?;
-                    print!("    - ");
+                    print!("{}- ", pad_spaces(4));
```
Comment (JoshuaBatty) [inline src/ops/fuelup_check.rs:63]:
```suggestion
                    println!("{:>4}- ", "");
```

This seems more idiomatic.

**Comment by JoshuaBatty** [comment]
Thanks for the PR @levisyin! This is certainly nicer than what is currently being done. 

Instead of using the `pad_spaces` function, perhaps we should use rust's built-in alignment capabilities as I suggested above. You can see more info about that [here](https://doc.rust-lang.org/std/fmt/index.html#fillalignment). Thanks!

**Comment by levisyin** [comment]
> Thanks for the PR @levisyin! This is certainly nicer than what is currently being done.
> 
> Instead of using the `pad_spaces` function, perhaps we should use rust's built-in alignment capabilities as I suggested above. You can see more info about that [here](https://doc.rust-lang.org/std/fmt/index.html#fillalignment). Thanks!

Hi sir, using rust's built-in alignment won't work well for issue #182 . Think about this:
I wanna print!("    -hello world"), with 4 space left on "-hello world". If using rust's built-in alignment, we should calculate the whole str length to find how many padding we should set. In this case, string length is 16, so we should set println!("{:>16}- ", ""). This is not convenient enough for us to quick set the num space we wanna leave.

**Comment by levisyin** [comment]
Done! See following creenshot:

image

**Review by JoshuaBatty — APPROVED** [review]
Nice, thanks for the PR!

**Pull request merged**